staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -139,7 +140,7 @@ func NewAuditOptions() *AuditOptions {
 				BatchConfig: pluginbuffered.NewDefaultBatchConfig(),
 			},
 			TruncateOptions:    NewAuditTruncateOptions(),
-			GroupVersionString: "audit.k8s.io/v1beta1",
+			GroupVersionString: "audit.k8s.io/v1",
```

Comment:
Does this mean we call out using v1 objects by default after this? Do we have an upgrade policy for this kind of changes?

Reply:
> Does this mean we call out using v1 objects by default after this?

Yes.

> Do we have an upgrade policy for this kind of changes?

I remembered that we don't support group version when upgrade from alpha to beta.
Or we still use v1beta1 here, and use v1 in the next release?

Reply:
> Or we still use v1beta1 here, and use v1 in the next release?

Yes, this is what I had in mind.

Reply:
done.